src/utils/rest-api.js

```diff
@@ -723,3 +723,41 @@ export function getLoadFlowParameters(studyUuid) {
             : response.text().then((text) => Promise.reject(text))
     );
 }
+
+function changeLineStatus(studyUuid, lineId, status) {
+    const changeLineStatusUrl =
+        getStudyUrl(studyUuid) +
+        '/network-modification/lines/' +
+        encodeURIComponent(lineId) +
+        '/status';
+    console.debug('%s with body: %s', changeLineStatusUrl, status);
+    return backendFetch(changeLineStatusUrl, { method: 'put', body: status });
+}
+
+export function lockoutLine(studyUuid, lineId) {
+    console.info('locking out line ' + lineId + ' ...');
+    return changeLineStatus(studyUuid, lineId, 'lockout');
```

Comment:
An enum for status ?

Reply:
I don't think it will be useful to use an enum here, this wouldn't prevent for mistakes.